api/models/Event.js

```diff
@@ -0,0 +1,46 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const OPERATION_CHECK_IN = 'check in';
```

Comment:
this could have less repetition by:
```
const operations = {
  OPERATION_CHECK_IN: 'check in',
  OPERATION_CHECK_OUT: 'check out',
  ...
};
const {} = operations
```
you could then in turn use the operations in the enum as:
`Object.keys(operations).map(key => operations[key]);`

And if you need to export the `const`s too, you can do that like:
`Object.assign(model,operations)`

Which ultimately would save you having to add/maintain an event in 3 places